docs/migration_guide.rst

```diff
@@ -154,7 +154,7 @@ This version deprecates several modules and fixes underlying issues with several
 0.14.0 Deprecations
```

Comment:
Perhaps lets already start including 0.16.0 removals in the migration guide as part of these PRs?